docs/userguide.md

```diff
@@ -174,6 +175,13 @@ Status:
     Status:                True
     Type:                  Ready
 ```
+### Unjoin Clusters
+
+If required, federation allows you to unjoin clusters using `kubefed2` tool.
+
+```bash
+kubefed2 unjoin cluster2 --host-cluster-context cluster1
```

Comment:
How about add some comments as `Join Clusters`?

```
You can repeat above step to un-join any additional clusters.
```

Reply:
Sure I will add it